smac/tae/execute_ta_run.py

```diff
@@ -35,6 +35,18 @@ def enum_hook(obj):
                 return getattr(globals()[name], member)
         return obj
 
+class BudgetExhaustedException(Exception):
+    """ Exception indicating that time- or memory-budgets are exhausted. """
+    pass
+
+class TAEAbortException(Exception):
+    """ Exception indicating that the TAE suggests an ABORT of SMAC. """
```

Comment:
@mlindauer is this correct? It can be the TAE (in case of the first run crashing, but there's a special exception for that), but mostly this is supposed to be sent by the target algorithm, right? Also, I'm not sure if the user should be confronted to the abbreviation TAE.